lora/lorawan/region/au915.go

```diff
@@ -0,0 +1,22 @@
+package region
+
+import "tinygo.org/x/drivers/lora"
+
+type RegionSetAU915 struct {
```

Comment:
Maybe this type should be named `SettingsAU915` making it `region.SettingsAU915` from outside.